twitter/View Controllers/LoginViewController.m

```diff
@@ -29,8 +29,6 @@ - (IBAction)didTapLogin:(id)sender {
     [[APIManager shared] loginWithCompletion:^(BOOL success, NSError *error) {
         if (success) {
             [self performSegueWithIdentifier:@"loginSegue" sender:nil];
-        } else {
```

Comment:
This is good (because we are removing the nslog here), but it would be great to have some error handling. Nothing actionable right now, but let's think about this